test/repository_test.rb

```diff
@@ -82,6 +82,30 @@
             assert updated_user.updated_at > @updated_at
           end
         end
+
+        describe 'when undefined method for Class' do
```

Comment:
What do you think about improving this test case message? To me, undefined method is too vague, what method that are undefined? What is Class?